pre_commit/runner.py

```diff
@@ -43,17 +44,39 @@ def repositories(self):
             repository.require_installed()
         return repositories
 
+    def get_hook_path(self, hook_type):
+        return os.path.join(self.git_root, '.git', 'hooks', hook_type)
+
     @cached_property
     def pre_commit_path(self):
-        return os.path.join(self.git_root, '.git', 'hooks', 'pre-commit')
+        return self.get_hook_path('pre-commit')
+
+    @cached_property
+    def pre_push_path(self):
+        return self.get_hook_path('pre-push')
+
+    @cached_property
+    def pre_template(self):
+        return resource_filename('hook-tmpl')
 
     @cached_property
+    def pre_push_template(self):
+        return resource_filename('pre-push-tmpl')
```

Comment:
these don't really belong in the runner, I'd just keep these in the place that installs